src/components/CreateRules/UploadRulesSummary.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <v-card flat>
+    <!-- Summary Section -->
+    <div id="upload-rules-summary" v-if="isSummary">
```

Comment:
Since this component is only used for summary purposes, you don't need isSummary. It complicates things unnecessarily.

Reply:
removed `v-if="isSummary"`